src/Route.php

```diff
@@ -73,7 +73,7 @@ public function hasDispatcher(): bool
      */
     public static function get(string $pattern, ?MiddlewareDispatcher $dispatcher = null): RouteInterface
     {
-        return self::methods([Method::GET], $pattern, $dispatcher);
+        return self::methods([Method::GET, Method::HEAD], $pattern, $dispatcher);
```

Comment:
From https://github.com/nikic/FastRoute#a-note-on-head-requests:

> Finally, note that applications MAY always specify their own HEAD method route for a given resource to bypass this behavior entirely.

That's an interesting point for the following code:

```php
Route::get('/bla', 'getBlaHandler');
Route::head('/bla', 'getBlaHeadHandler')
```

With the change in this PR we won't be able to do it, I think. But that might be a non-issue...

Reply:
Likely should be done in fastroute adapter instead.